en/docs/develop/advanced-development/applying-security-to-an-api.md

```diff
@@ -68,18 +70,18 @@ public class BasicAuthHandler implements Handler {
                 }
             }
         }
-        return false;
+        return true;
```

Comment:
Why is this changed? The default return value should be false.

Reply:
Yeah, it was a mistake while copy. Corrected.